10k validation changes

priyamtejaswin opened the pull request:
- Diff only from relevant 10k images
- `correct_index` is the position of the correct image for a caption, which is the iterator(index) value

@akshaychawla please verify the logic!

akshaychawla committed:
display only number of correct captions

akshaychawla committed:
validation script; 10k captions
- Sample any 10k captions instead of 40k captions for checking
top_K validation accuracy.
- top_K -> K = 100

priyamtejaswin committed:
validation_script.py | Updated validation logic.

ptejaswin commented on validation_script.py:
```diff
@@ -100,19 +101,25 @@ def on_epoch_end(self, epoch, logs={}):
 
-			correct_index = just_indices[i] 
-			if correct_index in top_k_indices:
+			correct_index = i ## The image at the ith position is true, since the diff is from im_outs_10k
```
**i** is from the original range of 0 to 39,999.
We should be using the *position* of **i** in that loop.
Good find @akshaychawla !
Fixing it now.

priyamtejaswin committed:
i is from the original range of 0 to 39,999.
We should be using the position of i in that loop.
Good find @akshaychawla !
Fixing it now.
